### PR TITLE
chore(infrastructure): update dependencies 

### PR DESCRIPTION
**Pull request by ghalibansari**

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -40,6 +40,17 @@
       "contributions": [
         "code"
       ]
+    },
```
Comment (Goldziher) [inline .all-contributorsrc:43]: Rollback this change. This is not done manually but rather by command. Google it to understand

**Comment by Goldziher** [comment]
@all-contributors add @ghalibansari code infrastructure

**Comment by ghalibansari** [comment]
> @all-contributors please add @ghalibansari for code infrastructure

not sure but i think this didnt worked.

**Comment by Goldziher** [comment]
> > @all-contributors please add @ghalibansari for code infrastructure
> 
> not sure but i think this didnt worked.

It didn't, I'll fix it when I'm bk from vacation

**Pull request merged**